src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    private static final FigureSupplire figureSupplire = new FigureSupplire();
+
+    public static void main(String[] args) {
+        Figure[] figure = new Figure[6];

```

Comment:
Consider using a more descriptive variable name for the 'figure' array. For example, 'figures' would be more appropriate as it indicates a collection of figures. 🧩